### ISSUE TITLE
Does Inference Engine means pre trained models trained on benchmark datasets like MS COCO are exposed directly for inference?

### ISSUE THREAD

**Issue by shubh1608** [Tessellate-Imaging/Monk_Object_Detection#75]
I am looking to use standard pre-trained models, my requirement is to compare different Image classification and Object detection pre-trained models, last time when I checked I got to know that the inference using pre-trained models directly is yet to be enabled.

I see in the doc about Inference Engine, seems like standard pre-trained models are exposed and can be used directly for inference. Is this right? If yes please guide me to some documents/examples on how to use this feature.

**Comment by abhi-kumar** [comment]
Hi, yes the inference engine is being developed for the purpose of using pretrained models (developed by original authors of the pipelines/algorithms) for users to check on their datasets for runtime and accuracy metrics.

Check these notebooks for examples - https://github.com/Tessellate-Imaging/Monk_Object_Detection/tree/master/example_notebooks/inference_engine

**Note: Do check the license of the original pipeline and the dataset the model is trained upon to use the inference code and the models in your project, monk inference engine is just a low code wrapper, usage restrictions are still applied as per the original implementations**

More Tutorials are in development.

**Comment by shubh1608** [comment]
Thanks @abhi-kumar for the quick update.
